dependabot.yaml

```diff
@@ -0,0 +1,8 @@
+version: 2
+updates:
+
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      # Check for updates to GitHub Actions every month
+      interval: "monthly"
```

Comment:
I wish GH supported something between "weekly" and "monthly".

Reply:
Unfortunately the [only options](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#scheduleinterval) are "daily", "weekly", and "monthly". I think this ends up fine though. Based on experience with past projects that have used this, there really aren't too frequent of updates.